hw/top_earlgrey/data/ip/chip_entropy_src_testplan.hjson

```diff
@@ -61,9 +74,122 @@
+      ]
       stage: V2
+      si_stage: SV3
+      lc_states: ["TEST_UNLOCKED", "PROD"]
       tests: ["chip_sw_entropy_src_kat_test"]
+      bazel: []
+    },
+    {
+      name: chip_sw_entropy_src_bypass_mode_health_tests
+      desc: '''
+            Verify that ENTROPY_SRC:
+            - provides one initial seed in boot-time / bypass mode only,
+            - signals a recoverable alert if the total failure counter hits the threshold in boot-time / bypass mode.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure loose health test threshold values for boot-time / bypass mode.
+            - Enable ENTROPY_SRC in boot-time / bypass mode.
+            - Enable CSRNG.
+            - Enable both EDNs in boot-time request mode.
+            - Trigger the execution of a cryptographic hardware block consuming entropy (e.g. AES) to test EDN0.
+            - Verify the entropy consuming endpoint finishes its operation.
+            - Trigger the execution of an OTBN program requiring entropy from both EDN1 and EDN0.
+            - Verify the OTBN program hangs.
+            - Disable the entropy complex.
+            - Configure unrealistically stringent health test threshold values for boot-time / bypass mode.
+            - Configure an alert threshold value of 1 in the ALERT_THRESHOLD register.
+            - Enable ENTROPY_SRC in boot-time / bypass mode.
+            - Enable CSRNG.
+            - Enable both EDNs in boot-time request mode.
+            - Verify that ENTROPY_SRC triggers a recoverable alert and sets the RECOV_ALERT_STS.ES_MAIN_SM_ALERT bit.
+            '''
+      features: [
+        "ENTROPY_SRC.MODE.BYPASS",
+        "ENTROPY_SRC.HEALTH_TESTS",
+      ]
+      stage: V3
+      si_stage: SV3
+      tests: []
+      bazel: []
+    },
+    {
+      name: chip_sw_entropy_src_fips_mode_health_tests
+      desc: '''
+            Verify that ENTROPY_SRC:
+            - provides continuous entropy in FIPS/CC compliant mode,
+            - works with different health test window sizes in FIPS mode,
+            - signals a recoverable alert and stops serving entropy if the total failure counter hits the threshold in FIPS mode.
+
+            Procedure:
+            - Disable the entropy complex.
+            - Configure realistic health test threshold values for FIPS/CC compliant mode.
+            - Enable ENTROPY_SRC in FIPS mode.
+            - Enable CSRNG.
+            - Enable both EDNs in auto request mode.
+            - Trigger the execution of various cryptographic hardware blocks consuming entropy to stress test the entropy complex.
+            - Verify that all entropy consuming endpoints can finish their operations and do not hang.
+            - Repeat the procedure with different health test window sizes for FIPS mode.
+            - Disable the entropy complex.
+            - Configure very stringent health test threshold values for FIPS mode.
```

Comment:
Suggestion: replace "very" by "unrealistically" to align with wording above

Reply:
Done, thanks.